File: Selenium/Activities/Activity1.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=service) as driver:
    driver.get("https://v1.training-support.net/")
    print("Home Page Title: ", driver.title)
    driver.find_element(By.ID, "about-link").click()
    print("About Us Page Title: ", driver.title)

File: Selenium/Activities/Activity1.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[] args)
    {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriverManager.firefoxdriver().setup();
        FirefoxDriver driver=new FirefoxDriver();
        driver.get("https://v1.training-support.net/");
        System.out.println("Home Page Title: " + driver.getTitle());
        driver.findElement(By.id("about-link")).click();
        System.out.println("About Us Page Title: "+driver.getTitle());
        driver.close();

    }

}
